INSTALL.md

```diff
@@ -332,3 +332,5 @@ For the competition you're working with, run the instructions from
       probably don't want that to be the donor's first experience.
 
 Please add items to this list as needed.
+
+If you have not set up 3rd party login, you can access [local login as well](https://github.com/OpenTechStrategies/torque-sites/blob/main/DESIGN.md#local-login-ie-regular-non-okta-mediawiki-login-is-also-possible)
```

Comment:
"instead" instead of "as well"

(You don't know how tempted I was to write "'instead' as well as 'as well'".  No, wait, scratch that: you *do* know.)